tasks/sanity-optional-label-check.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sanity-optional-label-check
+spec:
+  params:
+    - name: sourceImage
+      description: the fully qualified image name
+      default: quay.io/cvpops/test-index:v4.10
```

Comment:
I would avoid using default when we really want to change it.

Reply:
done